packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -233,80 +233,87 @@ const makeRenderer = (
 	const host = `${requestProtocol}://${domain}`;
 	const userAgent = headers['user-agent'];
 	const userAgentDevice = headers['x-ua-device'] || ''; // set by fastly
+	const memberId = parseMemberCookie(state).id;
```

Comment:
what if a member isn't logged in? should we have a fallback anonymous id

Reply:
`parseMemberCookie` handles it